include/map.h

```diff
@@ -210,19 +210,35 @@ typedef struct EnemyTerritoryThing {
     /* 0x1C */ s16 unk_1C;
 } EnemyTerritoryThing; // size = 0x20
 
-typedef struct EnemyTerritory {
-    /* 0x00 */ Vec3i unk_00;
-    /* 0x0C */ char unk_0C[0x8];
-    /* 0x14 */ s32 unk_14;
-    /* 0x18 */ s32 unk_18;
-    /* 0x1C */ s32 pointX;
-    /* 0x20 */ s32 pointY;
-    /* 0x24 */ s32 pointZ;
-    /* 0x28 */ s32 sizeX;
-    /* 0x2C */ s32 sizeZ;
-    /* 0x30 */ s32 shape;
-    /* 0x34 */ s32 unk_34;
-} EnemyTerritory; // size = ???
+typedef struct {
+    /* 0x00 */ Vec3i point;
+    /* 0x0C */ s32 wanderSizeX;
+    /* 0x10 */ s32 wanderSizeZ;
+    /* 0x14 */ s32 moveSpeedOverride;
+    /* 0x18 */ s32 unk_18; // 0 if WanderCircle, 1 otherwise
```

Comment:
Name this `wanderShape` and give it the type `enum { SHAPE_CYLINDER, SHAPE_RECT }`